.eslintrc.js

```diff
@@ -30,5 +34,13 @@ module.exports = {
         'jsx': 'never'
       },
     ],
+    'testing-library/no-debug': 'error',
```

Comment:
All the optional rules from:
https://github.com/testing-library/eslint-plugin-testing-library#supported-rules